pkg/constants/chain.go

```diff
@@ -15,34 +14,14 @@ const (
 	HUOBI   Chain = "heco"
 )
 
-const (
-	PS_GetPriceWithAddress    APIName = "v1/pricestore/chain/:chain/:address"
-	PS_GetTokensPrice         APIName = "v1/pricestore/chain/:chain/tokens"
-	PS_GetLpTokenPrice        APIName = "v1/pricestore/chain/:chain/lptokens"
-	PS_GetLosers              APIName = "v1/pricestore/chain/:chain/losers"
-	PS_GetGainers             APIName = "v1/pricestore/chain/:chain/gainers"
-	PS_GetPriceWithSymbol     APIName = "v1/pricestore/:symbol"
-	TS_GetDetailsWithContract APIName = "v1/tokenstore/token/address/:address"
-	TS_GetTokenList           APIName = "v1/tokenstore/token/all"
-	TS_GetTokenWithSymbol     APIName = "v1/tokenstore/token/symbol/:symbol"
-	ASSETS_GetAssets          APIName = "v1/:chain/address/:address/assets"
-	NFT_GetAssets             APIName = "v1/:chain/address/:address/nft-assets"
-	NFT_GetTxns               APIName = "v1/:chain/address/:address/nft-transactions"
-	NFT_GetDetailsWithID      APIName = "v1/:chain/address/:address/details"
-	NFT_GetHoldersByID        APIName = "v1/:chain/address/:address/nftholders"
-	TXN_GetTokenTxns          APIName = "v1/:chain/address/:address/transactions"
-	TXN_GetTxnDetails         APIName = "v1/:chain/transactions/:txnID"
-	TXN_GetTokenTxnsV2        APIName = "v2/:chain/address/:address/transactions"
-)
-
 //This should be manually changed when a new chain starts being supported
 var allChains = map[Chain]bool{ETH: true, BSC: true, MATIC: true, XDC: true, SOL: true, ZILLIQA: true, HUOBI: true}
 
 var priceStoreSupported = map[Chain]bool{ETH: true, BSC: true, MATIC: true}
 
 var nftEVMSupport = map[Chain]bool{ETH: true, BSC: true, MATIC: true}
 
-var allowedCallersByAPI = map[APIName]map[Chain]bool{
+var allowedCallersByChain = map[APIName]map[Chain]bool{
```

Comment:
you could move this to the api_name file as all its uses are present there and the key of the map is also api_name